src/arrayReverse.test.js

```diff
@@ -8,13 +8,62 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(arrayReverse([''])).toBeInstanceOf(Array);
+  });
 
+  it('should return an empty string '
+    + 'if original array consists of an empty string', () => {
+    expect(arrayReverse(['', '']))
+      .toEqual(['', '']);
   });
 
```

Comment:
you should have two checks: 1 - if original array is empty; 2 - if original array consists of an empty string